mcs/class/System/Mono.UnityTls/UnityTls.cs

```diff
@@ -176,6 +177,8 @@ public class unitytls_interface_struct
 
             public delegate unitytls_key_ref                              unitytls_key_get_ref_t(unitytls_key* key, unitytls_errorstate* errorState);
             public unitytls_key_get_ref_t                                 unitytls_key_get_ref;
+            public delegate unitytls_key*                                 unitytls_key_parse_pem_t(Int8* buffer, size_t bufferLen, UInt8* password, size_t passwordLen, unitytls_errorstate* errorState);
```

Comment:
If Int8* is supposed to indicate this is a char* pointer you may want to change password to also use Int8*